Sonify/index.html

```diff
@@ -97,17 +94,16 @@
         </div>
     </div>
     <div id="footer-container">
-        <img id="info-icon" src="./images/Icon-round-Question_mark.svg" alt="Information" title="Information" draggable="false" @click="openInfoPage">
+      <span id="info-icon" title="Information" @click="openInfoPage">&#9432;</span>
     </div>
 </div>
-<script type="text/javascript" src="./lib/vue.min.js"></script>
+<script type="text/javascript" src="./node_modules/vue/dist/vue.min.js"></script>
```

Comment:
Alternatively, we could introduce a bundler (e.g., rollup or webpack) to allow module importing from `node_modules` into JS files directly. Or, we could link to CDN (though it might be less reliable).